doc/specs/#1043 - Set the initial position of the Terminal/spec.md

```diff
@@ -41,22 +41,22 @@ The steps of this process:
 
 1. Set the top-left origin, width and height to CW_USEDEFAULT.
 2. Get the dpi of the nearest monitor; Load settings.
-3. From settings, find the user-defined initial position and launch mode. 
+3. From settings, find the user-defined initial position and launch mode.

```

Comment:
VS Code doesn't like trailing whitespace and it's too much effort to revert the removals, so...